src/inmanta/moduletool.py

```diff
@@ -81,6 +84,11 @@ def dict_constructor(loader, node):
     yaml.add_constructor(_mapping_tag, dict_constructor)
 
 
+class ModuleVersionException(CLIException):
+    def __init__(self, msg: str) -> None:
+        super().__init__(5, msg)
```

Comment:
I agree with @arnaudsjs here that passing `exitcode` as a keyword argument would be cleaner. As commented on slack I think `def __init__(self, *args, exitcode: int, **kwargs)` could achieve this. That said, I'm fine with keeping it like this too if you prefer, I'm mainly commenting this here because it might have gotten missed on slack due to the timing.

Reply:
ok, I'll change it